homeassistant/components/simplisafe/__init__.py

```diff
@@ -35,28 +36,38 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+ATTR_LIGHT_STATE = "light_state"
```

Comment:
Shouldn't this be a light entity ?

Reply:
Technically yes, but this one is tricky. The base station light doesn’t respond in real time; rather, you can think of it as “should the light be on or off the next time the station is armed?” Because it isn’t real-time, it doesn’t make sense to create a light entity – after toggling it, the user would have to arm the system to see the results.

(For reference, even though this behavior is strange, I want to include it for a user-reported use case where they want to be able to disable the light when they have guests [since the base station is in the guestroom].)

Reply:
Ah I understand now.

Reply:
Can we change the name to indicate that it's only when armed, maybe armed_light_state ?

Reply:
Yes sir!